src/components/pages/ParentHome/ParentCalendar.js

```diff
@@ -4,31 +4,66 @@ import 'antd/dist/antd.css';
 
 import { Calendar, Badge } from 'antd';
 
-function ParentCalendar() {
+function ParentCalendar(props) {
+  const { schedule } = props;
+
   function getListData(value) {
     let listData;
     switch (value.date()) {
       case 8:
         listData = [
-          { type: 'warning', content: 'This is warning event.' },
-          { type: 'success', content: 'This is usual event.' },
+          {
```

Comment:
Looks good. This will be nice for the stakeholder to see. I'm curious how we could map through the tables to get their data to appear.

Reply:
Noice!  Pretty cool to see the calendar populating with the dummy data, nice work!